spikeextractors/baseextractor.py

```diff
@@ -271,11 +273,71 @@ def allocate_array(self, memmap, shape=None, dtype=None, name=None, array=None):
                 arr = np.zeros(shape, dtype=dtype)
         return arr
 
+    def set_annotation(self, annotation_name, value, overwrite=False):
```

Comment:
in neo we call annotate(self, **kargs):
and this support several annoation.

Reply:
So the thing I don't like about `**kwargs` is that it prevents using spaces..but I can call it annotate

Reply:
space for what ? for annotation name ?
You can, annotate({'my spaced key': v}) is this is really needed.

Reply:
I would keep the same signature as it is now